src/factories/Axis.ts

```diff
@@ -214,5 +219,17 @@ module n3Charts.Factory {
 
       return axis;
     }
+
+    cloneAxis(): D3.Svg.Axis {
```

Comment:
Yeah true. We use the class name `Axis` and the property `axis` which is a D3.Axis obejct. Hmm, maybe we can come up with a naming convention for D3 objects - maybe `d3Axis` or sth.

Currently I am also using following naming conventions for D3.JS: I prefix selections with `$` and joins with `$$`. Maybe we can also introduce this some day.